### PR TITLE
Request to merge Feature styling

### PR DESCRIPTION
**Pull request by Kingstalux**
- I added styling to the Bookstore App. 
- I used Zeplin as a collaboration tool while styling the book store app.
- A preview of the app has been added to the README file.
- A live link to the page has been added to the README file.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,277 @@
...
+  background-color: white;
+  color: black;
+  font-weight: 900;
+  margin-bottom: 0.25rem;
+  font-size: 1.813rem;
+}
+
+p.book-author {
+  background-color: white;
+  color: #0290ff;
```
Comment (Ghiftee) [inline src/index.css]: - [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,277 @@
...
+  font-size: 1.188rem;
+  margin-bottom: 1.313rem;
+}
+
+.btn {
+  color: #0290ff;
```
Comment (Ghiftee) [inline src/index.css]: - [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

File: src/index.css [hunk 3]
```diff
@@ -1,3 +1,277 @@
...
+}
+
+.icon svg {
+  color: #0290ff;
+  font-size: 2.249rem;
+  background-color: white;
+}
+
+.circle svg {
+  color: #0290ff;
```
Comment (Ghiftee) [inline src/index.css]: - [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

File: src/index.css [hunk 4]
```diff
@@ -1,3 +1,277 @@
...
+
+div.middle {
+  margin-left: 0.688rem;
+  padding-bottom: 3.875rem;
+  background-color: white;
+}
+
+p.current {
+  font-size: 1.125rem;
+  color: #888;
```
Comment (Ghiftee) [inline src/index.css]: - [x] This should have color #e8e8e8 as per the Zeplin design

File: src/index.css [hunk 5]
```diff
@@ -1,3 +1,277 @@
...
+}
+
+p.number {
+  font-size: 2.438rem;
+  padding-top: 3.188rem;
+}
+
+p.completed {
+  font-size: 1.125rem;
+  color: #888;
```
Comment (Ghiftee) [inline src/index.css]: - [x] This should have color #e8e8e8 as per the Zeplin design.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽 @Kingstalux 

Good job working on the project so far! 
✔️ Your design is neat
✔️ There are no broken images or links
✔️ Your linters pass
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] The add book button should be on the same line as the book title and category input as shown in the design. Please make the changes
- Your design
Screenshot 2021-11-18 at 12 16 59

- Zeplin design
Screenshot 2021-11-18 at 12 17 22

- [x] Your design currently misses the horizontal line between the completed percentage and the current chapter as shown in the image below. Also, the zeplin design has a half-circle but your design has a full one, kindly fix them

- Your design
Screenshot 2021-11-18 at 12 20 44

- Zeplin design
Screenshot 2021-11-18 at 12 20 40

- [x] Your design uses the Times font-family however, the Zeplin design uses Robotoslab and Montserrat. You can have a look at the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information on where to use which font family.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.css (hunk 4 above)
Comment (Kingstalux) [reply src/index.css]: okay, I've implemented all color changes.

**Comment by Kingstalux** [comment]
- Hello @Ghiftee, thanks for the feedback, it was very helpful. So I've applied the following changes.
- I've corrected the colors and font families for the elements according to the designs
- I've added the vertical line in between the percentage and the chapter.
- I have aligned the add button with the inputs but depending on your screen resolution it might not look aligned since my resolution might be either bigger or smaller than yours. So maybe check on that.
- Finally for the circle, I took it from React font awesome icons, and only a full circle is found there. I tried exporting the half-circle from Zeplin but to no avail.
- Thats all for now, thanks.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Kingstalux, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**